### DISCUSSION TITLE
Setting -ens_mean for EnsembleStat in METplus

### DISCUSSION THREAD

**Discussion by michelleharrold** [dtcenter/METplus#1567] [Configuration] [answered]
Hello!

I would like to provide ensemble mean output from gen-ens-prod as input to ensemble-stat via the -ens_mean command line option. I gave a cursory look at the METpus documentation, but I couldn't figure out how to do this (I am sure I am glazing over something, so apologies, if it is obvious!).

Thanks!
Michelle Harrold

**Comment by bikegeek** [comment, marked as answer]
Hi Michelle,

I created a Github issue on METplus for this:
Provide instructions on usage of gen-ens-prod -ens_mean command line option #1569

Please feel free to make any modifications to the ticket.
